### PR TITLE
Made the tags in announcement and news app non-mandatory

### LINKED ISSUE #104: Tags are mandatory in announcement and news app
While adding announcement or news we cannot leave the tags field empty. Is this a bug or it is intended to have a tag for each thing.
cc: @yashLadha

### PR DESCRIPTION
**Pull request by Monal5031**
Fixes #104

### CODE REVIEW

File: wsgi/odyssy/announcement/models.py [hunk 1]
```diff
@@ -55,6 +55,6 @@ def get_model_type():
     key = models.AutoField(primary_key=True)
     start_date = models.DateTimeField(default=utils.get_today_start)
     end_date = models.DateTimeField(default=utils.get_today_end)
-    tags = TaggableManager()
+    tags = TaggableManager(blank=True)
```
Comment (Prakash2403) [inline wsgi/odyssy/announcement/models.py]: I think `null = True` will also be required, if you want to store `null` values in database.

File: wsgi/odyssy/news/models.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ class Meta:
     start_date = models.DateTimeField(default=timezone.now)
     end_date = models.DateTimeField(default=timezone.now)
     title = models.CharField(max_length=50)
-    tags = TaggableManager()
+    tags = TaggableManager(blank=True)
```
Comment (Prakash2403) [inline wsgi/odyssy/news/models.py]: See above comment.

File: wsgi/odyssy/announcement/models.py (hunk 1 above)
Comment (Monal5031) [inline wsgi/odyssy/announcement/models.py]: okay 👍

**Comment by Monal5031** [comment]
@yashLadha sir adding a default tag maybe lead to undesired things in future. your views?
As two different announcement or news will have the same default tag if not provided.

File: wsgi/odyssy/announcement/models.py (hunk 1 above)
Comment (Monal5031) [inline wsgi/odyssy/announcement/models.py]: @Prakash2403 sir there is no **kwargs of null in TaggableManager().

**Comment by yashLadha** [comment]
There can be. It's not a problem as some academic announcements will have academic tag and they can be more than two. Tags will only work for sorting purpose. It is better to give them something them to give them nothing. That way it would be more consistent as all announcements and news will have some tag with them.

**Comment by Monal5031** [comment]
Okay then I'll add `announcement` as default tag for announcement and `news` for news.

**Comment by Monal5031** [comment]
@yashLadha Sir please review, I think this should work.

File: wsgi/odyssy/news/models.py [hunk 3]
```diff
@@ -61,3 +61,12 @@ def get_latest_news_tag(tag_name, cnt):
     @staticmethod
     def get_model_type():
         return "News"
+
+
+def set_default_tag(sender, instance, **kwargs):
+    post_save.disconnect(set_default_tag, sender=sender)
+    instance.tags.add('news')
```
Comment (yashLadha) [inline wsgi/odyssy/news/models.py]: Add tag's only if user skips adding the tag.

Comment (Monal5031) [inline wsgi/odyssy/news/models.py]: Oh, sorry missed the condition.

**Comment by Monal5031** [comment]
@yashLadha sir I've made the changes please take a look.

**Comment by yashLadha** [comment]
@Prakash2403 Update your review.

**Pull request merged**